tests/cypress/integration/components/a-z-nav.js

```diff
@@ -0,0 +1,46 @@
+describe('setting screen res', function () {
+
+  beforeEach(function () {
+    cy.viewport('macbook-15');
+
+  });
+
+  it('Navigating to A-Z Navigation', function () {
+
+    cy
```

Comment:
Can this go on the same line `cy.visit()`?

Reply:
removed the spacing